example_configs/bootstrap/bootstrap.md

```diff
@@ -205,14 +264,15 @@ spec:
           volumeMounts:
             - name: bootstrap
```

Comment:
Why not mount the whole bootstrap directory? That was the point of the change, no?

Reply:
I was thinking about it, but I've never setup kubernetes. Below there are `volumes.projected.sources` . Someone who is able to test this config should have a look.